packages/opentelemetry-core/src/trace/suppress-tracing.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Context, createContextKey } from '@opentelemetry/api';
+
+const SUPPRESS_TRACING_KEY = createContextKey(
+  '@opentelemetry/core suppress.tracing'
```

Comment:
Not sure about the name here. specifying core somehow disallows moving just in case it will be added again in API.
The key names in API use a slight different schema.

Should we think about rules about context key names?

Reply:
I agree that i would prefer to use the same schema than API key

Reply:
the API does like `OpenTelemetry Context Key SPAN`, what about `OpenTelemetry SDK Context Key SUPPRESS_TRACING`?

Reply:
I'm not sure why we should change it all compared the old value ? I mean isn't it breaking if it isnt the same ?

Reply:
The old name had `SUPPRESS_INSTRUMENTATION`, when we are only suppressing tracing now so it doesn't really make sense. There are already many breaking changes requiring users to be on the latest SDK to use the next API release anyway.

Reply:
Agree, i think i misunderstood because you wrote in the PR comment

> Because we use Symbol.for to get keys, there is no version issue with this.

I though you were implying that this was backward compatible so yeah.

Reply:
No i meant that if multiple `core` versions are installed in nested `node_modules` they will get the same symbol because they use symbol.for